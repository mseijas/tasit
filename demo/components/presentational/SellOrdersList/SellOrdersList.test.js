import React from "react";
import { shallow } from "enzyme";
import SellOrdersList from ".";

describe("SellOrdersList", () => {
  it("renders the component", async () => {
    const renderItem = () => {};
    const sellOrders = [];
    expect(
      shallow(
        <SellOrdersList sellOrders={sellOrders} renderItem={renderItem} />
      )
    ).toMatchSnapshot();
  });
});
