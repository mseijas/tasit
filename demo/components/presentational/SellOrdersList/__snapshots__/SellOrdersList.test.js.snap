// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrdersList renders the component 1`] = `
<FlatList
  data={Array []}
  disableVirtualization={false}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  numColumns={1}
  onEndReachedThreshold={2}
  renderItem={[Function]}
  scrollEventThrottle={50}
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
  updateCellsBatchingPeriod={50}
  windowSize={21}
/>
`;
